CONTRIBUTING.md

```diff
@@ -10,8 +10,8 @@ Thank you for considering contributing to our Python project! By contributing, y
 
 ```bash
 git clone https://github.com/ohsu-comp-bio/vrs-python-testing
-python3 -m venv venv ; source venv/bin/activate
-pip install -r requirements.txt
+python3.10 -m venv venv; source venv/bin/activate
```

Comment:
We shouldn't need to overspecify 3.10
We should be running on >=3.10

Reply:
updated